bin/avc-test-targeted-attack

```diff
@@ -12,20 +12,28 @@ if __name__ == '__main__':
     parser.add_argument(
         "--no-cache", action='store_true',
         help="Disables the cache when building the attack image.")
+    parser.add_argument(
+        "--no-build", action='store_true',
+        help="Disables building the image, and assumes they exist. ")
     parser.add_argument(
         "--gpu", type=int, default=0, help="GPU number to run container on")
     parser.add_argument(
         "--samples", type=int, default=100,
         help="Number of samples for testing.")
+
     args = parser.parse_args()
+    cache = ''
+    no_build = ''
     if args.no_cache:
         cache = ' --no-cache'
-    else:
-        cache = ''
-    cmd = ('avc-test-attack{cache} --gpu {gpu} --mode targeted'
+    if args.no_build:
+        no_build = ' --no-build'
+
+    cmd = ('avc-test-attack{cache} {no_build} --gpu {gpu} --mode targeted'
            ' --samples {samples} {directory}')
     cmd = cmd.format(
         cache=cache,
+        no_build=no_build,
         gpu=args.gpu,
```

Comment:
```
File "/gpfs01/bethge/home/bveliqi/.local/bin/avc-test-untargeted-attack", line 37
    gpu=args.gpu,
      ^
```

I'm getting this error with `Python 3.4.5` after having installed this branch when running `avc-test-untargeted-attack --no-build .`

Reply:
@bveliqi : Are you sure this is the whole error ?

Reply:
@spMohanty: 
![image](https://user-images.githubusercontent.com/874483/42446316-130ee20e-8376-11e8-8c9e-f393cd4bf5f0.png)

Reply:
Okay thanks. I had tested it on py36.
Shouldnt be very difficult to fix this.

We should probably hook the repo up with travis, so that we can run some basic tests across all envs for all pull requests.